src/test/java/seedu/duke/commands/ListCommandTest.java

```diff
@@ -30,23 +30,25 @@ public void printWithoutWrapping() {
         inventory.getTrie().add(item2.getName().toLowerCase());
 
         String expectedOutput =
-                "+-----------------+--------------+----------+----------+-----------------+" + System.lineSeparator() +
-                        "| Name            | UPC          | Quantity | Price    | Category        |" +
+                "+-----------------+--------------+----------+----------+-----------------+-------+" +
                         System.lineSeparator() +
-                        "+-----------------+--------------+----------+----------+-----------------+" +
+                        "| Name            | UPC          | Quantity | Price    | Category        | Index |" +
```

Comment:
LGTM! but just a thought, would have an index in the first column be a better arrangement? Because by intuition, people would usually expect the numbering of table rows to be on the most lefthand side

Something like this:
 "| Index | Name            | UPC          | Quantity | Price    | Category        |" +

Reply:
it does make more sense to have the index in the first column as compared to having it at the end next to the category. updated accordingly.

![table_index_left](https://user-images.githubusercontent.com/88529907/229711171-60ea67a6-aeab-4e48-bef3-71a4e65e557c.PNG)